config/locales/en.yml

```diff
@@ -1552,7 +1552,7 @@ en:
         global_carryforward_html: <a href="%{global_carryforward_url}" target="_blank">Global Carryforward:</a> You allow us to make your tax return information available to other VITA programs you may visit.
         legal_details_title: The legal details
         legal_details_html: |
-          <p>Federal law requires these consent forms be provided to you. Unless authorized by law, we cannot disclose your tax return information to third parties for purposes other than the preparation and filing of your tax return without your consent. If you consent to the disclosure of your tax return information, Federal law may not protect your tax return information from further use or distribution.</p>
+          <p class="spacing-below-25">Federal law requires these consent forms be provided to you. Unless authorized by law, we cannot disclose your tax return information to third parties for purposes other than the preparation and filing of your tax return without your consent. If you consent to the disclosure of your tax return information, Federal law may not protect your tax return information from further use or distribution.</p>
```

Comment:
this class looks weird i know but i couldn't figure out how else to get rid of the bottom margin on just one of the p tags within the reveal (the only one i wanted to remove the margin from wraps the reveal link, but we don't have access to it)